content/en/docs/components/katib/experiment.md

```diff
@@ -41,159 +42,166 @@ the experiment from the UI or from the command line.
+- **resumePolicy**: Experiment resume policy. If experiment was succeeded because `maxTrialCount`
   was reached, you can resume it by increasing `maxTrialCount`. Specify `resumePolicy: LongRunning`, if
   you want to use this feature. If you don't need to resume experiment, specify `resumePolicy: Never`. In that case,
   suggestion resources will be deleted and experiment can't be resumed. By default all experiments have
-  `resumePolicy: LongRunning` parameter. See the [`ResumePolicy` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1alpha3/experiment_types.go#L53).
+  `resumePolicy: LongRunning` parameter.
+  See the [`ResumePolicy` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1beta1/experiment_types.go#L54).
 
-*Background information about Katib's `Experiment` type:* In Kubernetes 
+_Background information about Katib's `Experiment`, `Suggestion` and `Trial` type:_ In Kubernetes
 terminology, Katib's
-[`Experiment` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1alpha3/experiment_types.go#L211) is a [custom resource 
+[`Experiment` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1beta1/experiment_types.go#L269),
+[`Suggestion` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/suggestions/v1beta1/suggestion_types.go#L117) and
+[`Trial` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/trials/v1beta1/trial_types.go#L122) is a [custom resource
 (CR)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
 The YAML file that you create for your experiment is the CR specification.
 
 <a id="search-algorithms"></a>
-### Search algorithms in detail 
-  
+
+### Search algorithms in detail
+
 Katib currently supports several search algorithms. See the [`AlgorithmSpec`
-type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1alpha3/common_types.go#L23-L33).
+type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1beta1/common_types.go#L23-L33).
 
 Here's a list of the search algorithms available in Katib. The links lead to
 descriptions on this page:
 
-* [Grid search](#grid-search)
-* [Random search](#random-search)
-* [Bayesian optimization](#bayesian)
-* [Hyperband](#hyperband)
-* [Tree of Parzen Estimators (TPE)](#tpe-search)
-* [Covariance Matrix Adaptation Evolution Strategy (CMA-ES)](#cmaes)
-* [Neural Architecture Search based on ENAS](#enas)
-* [Differentiable Architecture Search (DARTS)](#darts)
+- [Grid search](#grid-search)
+- [Random search](#random-search)
+- [Bayesian optimization](#bayesian)
+- [Hyperband](#hyperband)
+- [Tree of Parzen Estimators (TPE)](#tpe-search)
+- [Covariance Matrix Adaptation Evolution Strategy (CMA-ES)](#cmaes)
+- [Neural Architecture Search based on ENAS](#enas)
+- [Differentiable Architecture Search (DARTS)](#darts)
```

Comment:
Suggestion - we should add this list of available NAS algos to the beginning of the Katib ABOUT page somewhere. Then add "more algorithms are under development."

Reply:
Sure, I have added: https://github.com/kubeflow/website/blame/b946be02fb367f8e9c2fb5d88a7818ddd7c18ee8/content/en/docs/components/katib/overview.md#L20-L24. WDYT @8bitmp3 ?